src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1453,5 +1483,336 @@ public void SarifTransformerTests_ToCurrent_TwoResultsWithFixes()
 
             v2LogText.Should().Be(v2LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_CodeFlows()
+        {
+            string v1LogText =
+@"{
+  ""version"": ""1.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0""
+      },
+      ""logicalLocations"": {
```

Comment:
I don't think we need v1 `logicalLocations` to test v1 => v2 code flows.

Reply:
The locations have fullyQualifiedLogicalNames

Reply:
Sure, but why does v1 `logicalLocations` have to exist? You'll create v2 `logicalLocations` when you notice the FQLNs in the code flow.

Reply:
The spec says that _if_ `run.logicalLocations` exists, _then_ `location.FQLN` must match one of its property names. But the spec does not say that `run.logicalLocations` has to exist.

Reply:
I'm sorry, I'm sure we went through this once before, but doesn't the presence of a FQLN imply a corresponding v1 logicalLocation?

Reply:
No. If you want to tell the consumer more about the logical location whose FQLN is `location.FQLN`, you can create the `run.logicalLocations` dictionary,  add a property whose name is the value of `FQLN`, and set the property value to a `logicalLocation` object with the additional information. But you didn't have to do it in v1, and you still don't have to do it in v2.

Reply:
(Don't worry about asking the same question multiple times. I'm notorious for doing that -- just ask Michael 😄.)

Reply:
Clear, thanks